frontend/ui/widget/radiobuttontable.lua

```diff
@@ -147,7 +147,11 @@ function RadioButtonTable:addHorizontalSep(vspan_before, add_line, vspan_after,
 end
 
 function RadioButtonTable:onSelectByKeyPress()
-    self:getFocusItem().callback()
+    local item = self:getFocusItem()
+    if item then
+        item.callback()
+    end
+    return item
```

Comment:
Why do you make many event handlers return the item (rather than true/false, what I was expecting).
Is this returned item used by some bits of this PR I didn't managed to see?

Reply:
I am a backend developer. I didn't learn lua before. So I read the book "Programming in Lua" first. It says

> In Lua, any value may represent a condition. Conditionals (such as the ones in control structures) consider false and nil as false and anything else as true.

So I just return item here. Should I use `item == nil`?

Reply:
Yes, returning `item` would work just as returning `true` (and normally, we prefer just using `item` to using `item == nil` in conditions, as the Lua book explains).
But for all onSomething even handlers, we (KOReader devs) are used to see `return true` (meaning propagation stopped) or `return false` (or no return acting as nil/false) (propagation can continue).
With `return item`, I would wonder (as I did and asked :) if you were somehow doing something with the returned item.
We're more used to see for this case:

```lua
    if item then
        item.callback()
        return true -- ok, processed
    end
    return true -- not procesed, but stop propagation anyway, upper widget should not process it
```
or 
```lua
    if item then
        item.callback()
        return true -- ok, processed
    end
    -- just nothing, or:
    return false -- not procesed, something else might want o handle it
```

Your `return item` works indeed as the first example - but it's just a bit less explicite :)